frontend/src/components/shared/StreamlitMarkdown/StreamlitMarkdown.tsx

```diff
@@ -247,8 +251,46 @@ export function RenderedMarkdown({
     h6: CustomHeading,
     ...(overrideComponents || {}),
   }
-
-  const plugins = [remarkMathPlugin, remarkEmoji, remarkGfm]
+  const theme = useTheme()
+  const colors = theme.colors as typeof themeColors
+  const colorMapping = new Map(
+    Object.entries({
```

Comment:
nit: Are we using the same colors for dark & light theme? otherwise, we might need a check here via

https://github.com/streamlit/streamlit/blob/0715c1d34b5670fb19dac164db2896daa082782a/frontend/src/theme/utils.ts

Reply:
No, now we define in `baseTheme` light colours and dark in `darkTheme` 👍